stsci/__init__.py

```diff
@@ -1,11 +0,0 @@
-try:
-    # As long as we're using setuptools/distribute, we need to do this the
-    # setuptools way or else pkg_resources will throw up unncessary and
-    # annoying warnings (even though the namespace mechanism will still
-    # otherwise work without it).  Get rid of this as soon as
-    # setuptools/distribute is dead.
-    __import__('pkg_resources').declare_namespace(__name__)
-except ImportError:
-    pass
-
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
What is this `__path__` used for?

While we're at this, should we add a `__all__` and include `__version__` like other packages, @zacharyburnett ?

Reply:
Oh wait, this is namespace package level... `stsci.tools` has an extra `lib` folder. My head hurts.

Reply:
Yup 😅

Reply:
> What is this `__path__` used for?

I'm not sure but you removed it in stsci.tools in https://github.com/spacetelescope/stsci.tools/pull/144.

Reply:
I just dunno if the presence of `lib/` folder matters or not. I pinged the SCSB packaging experts to review. 🤞 

Must have been painful to smoke this out. Have a virtual beer. 🍺

Reply:
Yeah it wasn't great, it looked like the failure was from `stsci.tools` but you'd already removed this code from there so I couldn't figure out where it was happening. I'm also making PRs to `stsci.imagestats` and `stsci.stimage` in addition to the two I made already.

Reply:
oof, this namespace stuff with `stsci.` makes my head hurt as well. As far as I can tell this is not needed... ?

As far as `__version__`, that should be in the metadata (available via `importlib.metadata`), but if we do ever need to reference the version from within the source, `setuptools_scm` can automatically generate and install a version file for you if you add the configuration to https://github.com/spacetelescope/stsci.image/blob/master/pyproject.toml#L56
```diff
[tool.setuptools_scm]
+ version_file = "stsci/image/version.py"
```
then you can reference it like `from .version import __version__` or `from stsci.image.version import __version__`